### PR TITLE
HZ-2:add email&password check function

### PR DESCRIPTION
**Pull request by maggieAU1314**
add email&password check function

### CODE REVIEW

File: src/components/SignUp/SignUpInfo.tsx [hunk 1]
```diff
@@ -27,24 +29,68 @@ const SignUpInfoButton = styled(Button)({
...
     }
-})
+});
+
+const validationSchema = yup.object({
+    email: yup
+      .string('Enter your email')
+      .email('Enter a valid email')
+      .required('Email is required'),
+    password: yup
```
Comment (yuanningd) [inline src/components/SignUp/SignUpInfo.tsx:39]: the password check do not meet ticket specification

File: src/components/SignUp/SignUpInfo.tsx [hunk 2]
```diff
@@ -27,24 +29,68 @@ const SignUpInfoButton = styled(Button)({
...
 
 const SignUpInfo = () => {
+    const formik = useFormik({
+      initialValues: {
+        email: '',
+        password: '',
+      },
+      validationSchema: validationSchema,
+      onSubmit: (values) => {
+        alert(JSON.stringify(values, null, 2));
```
Comment (yuanningd) [inline src/components/SignUp/SignUpInfo.tsx:56]: What is this for?

File: src/components/SignUp/SignUpInfo.tsx (hunk 1 above)
Comment (maggieAU1314) [reply src/components/SignUp/SignUpInfo.tsx:39]: Modified password check according to card specification

File: src/components/SignUp/SignUpInfo.tsx (hunk 2 above)
Comment (maggieAU1314) [reply src/components/SignUp/SignUpInfo.tsx:56]: used for testing that time，deleted now.

**Pull request merged**